pkg/api/handlers/tests/signup_handler_test.go

```diff
@@ -64,58 +65,42 @@ func (suite *SignUpHandlerTestSuite) TestSignUpHandlerSuccess() {
 		"password": "123123123"
 	}`)
 
-	req := httptest.NewRequest(http.MethodPost, "/signup", bytes.NewBuffer(requestBody))
-	req.Header.Set(echo.HeaderContentType, echo.MIMEApplicationJSON)
-	rec := httptest.NewRecorder()
+	request := httptest.NewRequest(http.MethodPost, "/signup", bytes.NewBuffer(requestBody))
+	request.Header.Set(echo.HeaderContentType, echo.MIMEApplicationJSON)
+	recorder := httptest.NewRecorder()
 
-	suite.Server.ServeHTTP(rec, req)
-	var jsonRes common.AuthResponse
+	suite.Server.ServeHTTP(recorder, request)
+	var response common.AuthResponse
 
 	ur, err := model.FindByEmail(context.Background(), "fizi@gmail.com")
 	assert.NoError(t, err)
 
-	assert.Equal(t, http.StatusCreated, rec.Code)
-	assert.NoError(t, json.Unmarshal(rec.Body.Bytes(), &jsonRes))
-	assert.Equal(t, jsonRes.Email, ur.Email)
-	assert.Equal(t, jsonRes.FirstName, ur.FirstName)
-	assert.Equal(t, jsonRes.LastName, ur.LastName)
+	assert.Equal(t, http.StatusCreated, recorder.Code)
+	assert.NoError(t, json.Unmarshal(recorder.Body.Bytes(), &response))
+	assert.Equal(t, response.Email, ur.Email)
+	assert.Equal(t, response.FirstName, ur.FirstName)
+	assert.Equal(t, response.LastName, ur.LastName)
```

Comment:
Change the order so the expecte value is the second parameter and the actual value the third